### ISSUE TITLE
Clicking on Ark Logo return Uncaught Promise Error

### ISSUE THREAD

**Issue by davimello28** [ArkEcosystem/desktop-wallet#1614]
Clicking on Ark Logo returns an `Uncaught (in promise)` error.

```js
{
    name: "NavigationDuplicated",
    message: "Navigating to current location ("/") is not allowed", 
    stack: "Error↵  at new NavigationDuplicated ( [...]./node_modules/vue/dist/vue.esm.js:1862:26)"
}
```

**Expected Behavior**

No error

**Current Behavior**

Well, the error.

**Possible Solution**

I was able to reproduce it from all of the views (Wallets, Contacts...). Double-clicking the same page also returns this error. Just caught the error seems to be ok.

**Steps to Reproduce (for bugs)**

1. Just click on the Ark logo.

**Your Environment**

- Version used: Last `develop` pull.
- Environment name and version: Firefox 72.0.1, Node 12.12
- Operating System and version: Debian 10

**Comment by davimello28** [comment]
All of the `NavigationDuplicated` errors are fixed on the newer version of `vue-router` that uses `async/await` API that resolves even if the route fails. See [this](https://github.com/vuejs/vue-router/issues/2872) and [this](https://github.com/vuejs/vue-router/issues/2881#issuecomment-520554378).

Updating `vue-router` to `^3.1.0` and creating the catches will fix this issue.

**Comment by dated** [comment]
Fixed in #1719.
